src/Test.FunctionalTests.Sarif/Test.FunctionalTests.Sarif.csproj

```diff
@@ -49,7 +49,6 @@
     <PackageReference Include="FluentAssertions" Version="5.10.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0-preview-20220707-01" />
     <PackageReference Include="Moq" Version="4.13.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
```

Comment:
>System.Collections.Immutable [](http://example.com/codeflow?start=31&length=28)

I'm removing the dependency from the tests, since it will use the same version of the product. This will facilitate new updates in the future if required.